ibm/mas_devops/roles/suite_dns/README.md

```diff
@@ -121,6 +121,26 @@ Role Variables - IBM Cloud Internet Services DNS Integration
 - Environment Variable: `CIS_SUBDOMAIN`
 - Default: None
 
+### cis_enhanced_security
+Set this to true to enable the enhanced CIS DNS integration security. See below for details.
```

Comment:
Can you add something that explains what this actually gives the user .. e.g. a link to the CIS docs related to enhanced security, someone reading our docs should be presented a simple avenue to find out more about this option.